tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -192,7 +192,6 @@ describe('Test Self-Healing of resource modification when correctDrift option us
       // Update exising GitRepo by enabling 'correctDrift'
       cy.addFleetGitRepo({ repoName, correctDrift: 'yes', editConfig: true });
       cy.clickButton('Save');
-      cy.open3dotsMenu(repoName, 'Force Update');
```

Comment:
maybe we should keep this ono in place as it may speedup the resources recovery

Reply:
yes, I am also proposing same. We can keep `Force Update` here for this test only.